### PR TITLE
Observability docs for Launch

### PR DESCRIPTION
**Pull request by tristan-spaulding2**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/launch-queue-observability.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# Queue observability (beta)
+
+Launch provides an interactive dashboard for each queue, allowing ML engineers and MLOps teams to make effective use of their available hardware. 
+
+By exploring this tab, you can see when the queue was in heavy use or idle, visualize what workloads were running, and spot inefficient jobs.  For deeper analysis, the page links to the W&B experiment tracking workspace and to external infrastructure monitoring providers like Datadog, NVIDIA Base Command, or cloud consoles.
+
+:::info
+To turn on this capability, toggle on the `Launch observability` feature flag in your user settings.
+
+This feature requires W&B Weave, which is not available on some Customer-managed and Dedicated Cloud.  Contact your W&B representative to learn more.
```
Comment (gtarpenning) [inline docs/guides/launch/launch-queue-observability.md]: The dashboard requires Weave 1 in particular, not sure if thats relevant. And **must** be turned on in user settings in SaS

File: docs/guides/launch/launch-queue-observability.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+:::
+
+## Dashboard and plots
+
+Each queue now has a `Monitor` tab.   When you click on the tab, W&B will gather data over the last 7 days of activity for this queue.  Time ranges, grouping and filters can be controlled from the left panel.
```
Comment (gtarpenning) [inline docs/guides/launch/launch-queue-observability.md]: Data gathering is technically independent, maybe "gather" -> "display"

File: docs/guides/launch/launch-queue-observability.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+### Job status
+
+![](/images/launch/launch_obs_jobstatus.png)
+
+This plot shows how many jobs were running, pending, queued, or completed in each time interval.  
+
+This can be useful for identifying periods of idleness in the queue--in the case of fixed resources (e.g. a DGX BasePod), that might suggest an opportunity to run lower-priority pre-emptible jobs such as sweeps.  Meanwhile, with cloud resources, recurring bursts might suggest an opportunity to save money by reserving resources for particular times.
+
+Seeing a lot of  `Queued` items might indicate an opportunities to shift worklaoads to other queues, while a spike in failures can identify users who might need help with their job setup.
```
Comment (gtarpenning) [inline docs/guides/launch/launch-queue-observability.md]:
```suggestion
Seeing a many `Queued` items might indicate opportunities to shift workloads to other queues, while a spike in failures can identify users who might need help with their job setup.
```

File: docs/guides/launch/launch-queue-observability.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Select a range to show more details in the plot below, or Zoom to filter the entire page.
+
+### Queued time
+
+![](/images/launch/launch_obs_queuedtime.png)
+
+This plot shows, for every time period, the amount of time in minutes that jobs were queued.  For example, if there were 10 jobs queued in that period, waiting for an average of 30 minutes each, the plot would show 300 minutes.
```
Comment (gtarpenning) [inline docs/guides/launch/launch-queue-observability.md]: this is currently in seconds, but we can change that if you think jobs are going to sit in queued for a long time...

File: docs/guides/launch/launch-queue-observability.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+This plot can be colored by the `Grouping` control in the left bar--which can be particularly helpful for identifying which users and jobs are feeling the pain of scarce queue capacity.
+
+### Job runs
+
+![](/images/launch/launch_obs_jobruns2.png)
+
+
+This plot shows the start and end of every job executed in a time period, with distinct colors for each run.  This makes it easy to see at a glance what workloads the queue was processing at a given time.  
+
+Use the Select tool in the bottom right of the panel to brush over jobs to see more details in the table below.
```
Comment (gtarpenning) [inline docs/guides/launch/launch-queue-observability.md]:
```suggestion
Use the Select tool in the bottom right of the panel to brush over jobs to populate the table below.
```

**Review by gtarpenning — APPROVED** [review]

File: docs/guides/launch/launch-queue-observability.md (hunk 4 above)
Comment (tristan-spaulding2) [reply docs/guides/launch/launch-queue-observability.md]: I would actually update it to minutes in the product... things will be queued for a while...I will doc it as seconds here though.

**Review by ngrayluna — APPROVED** [review]
Did some word smithing. Will do a follow up in the future to mention how to access this feature (post Beta).

File: docs/guides/launch/launch-queue-observability.md (hunk 4 above)
Comment (ngrayluna) [reply docs/guides/launch/launch-queue-observability.md]: QQ: In my revision, I mentioned seconds. Is this still correct.

PS It'd be nice to have the units on plots...

**Pull request merged**